src/microservices/Microservices.DicomReprocessor/Execution/Processors/DicomFileProcessor.cs

```diff
@@ -73,7 +73,8 @@ public void ProcessDocument(BsonDocument document)
             var message = new DicomFileMessage
             {
                 NationalPACSAccessionNumber = (string)headerDoc["NationalPACSAccessionNumber"],
-                DicomFilePath = (string)headerDoc["DicomFilePath"]
+                DicomFilePath = (string)headerDoc["DicomFilePath"],
+                DicomFileSize = (long)headerDoc["DicomFileSize"]
```

Comment:
Will this not explode as soon as we try and reconstruct messages which have not been created with this field already?

Reply:
Ooops yes, that seems likely.

Reply:
Should be fixed by 
https://github.com/SMI/SmiServices/commit/51c416a452c1247145bd4162fd25152169c42836